src/transformers/modeling_flax_bert.py

```diff
@@ -0,0 +1,365 @@
+from typing import Callable, Dict
+
+import numpy as np
+
+import flax.linen as nn
+import jax
+import jax.numpy as jnp
+from flax.linen import compact
+from transformers import BertConfig
+from transformers.modeling_flax_utils import FlaxPreTrainedModel
+
+
+@jax.jit
+def gelu(x):
+    r"""Gaussian error linear unit activation function.
+
+    Computes the element-wise function:
+
+    .. math::
+      \mathrm{gelu}(x) = \frac{x}{2} \left(1 + \mathrm{tanh} \left(
+        \sqrt{\frac{2}{\pi}} \left(x + 0.044715 x^3 \right) \right) \right)
+
+    We explicitly use the approximation rather than the exact formulation for
+    speed. For more information, see `Gaussian Error Linear Units (GELUs)
+    <https://arxiv.org/abs/1606.08415>`_, section 2.
+    """
+    return x * 0.5 * (1.0 + jax.lax.erf(x / jnp.sqrt(2.0)))
+
+
+ACT2FN = {
+    "gelu": nn.gelu,
+    "relu": nn.relu,
+    "swish": nn.swish,
+    "gelu_new": gelu,
+}
+
+
+class BertLayerNorm(nn.Module):
```

Comment:
maybe better call this `FlaxBertLayerNorm` ? To avoid having the same name in PT

Reply:
Done w.r.t @LysandreJik comment's